core-js/src/main/javascript/cdf/components/DateInputComponent.js

```diff
@@ -47,7 +47,9 @@ define([
 
       myself.placeholder()
         .addClass('date-input-container')
-        .html('<input class="date-input" id="' + inputId + '" value="' + inputValue + '"/>');
+        .html('<input class="date-input" id="' + inputId + '"/>');
+
+      $('#' + inputId).val(inputValue);
```

Comment:
Using a selector to search for an element that is being created here seems to be excessive and might affect performance.

Reply:
@afrjorge we can't do this in another way. And we do absolutely the same for text input component

Reply:
I don't understand. Is there any reason for the following not to work?
```javascript
var $el = $('<input class="date-input" id="' + inputId + '"/>');
$el.val(inputValue);

myself.placeholder()
  .addClass('date-input-container')
  .html($el);
```